index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Page</title>
+</head>
+<body>
+    <header id="top-navigation">
+        <div class="profile">
+            <a class="name" href="https://www.linkedin.com/in/patrick-ndung-u-251a1518b?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_view_base_contact_details%3B2E90C86VT4m5KKL5YVoCXg%3D%3D">Patrick Maina</a>
+        </div>
+        <nav id="side-nav">
+            <a class="fa fa-bars" href="#"></a>
+            <div class="portfolio">
+                <a href="#">Portfolio</a>
+            </div>
+            <div class="about">
+                <a href="#">About</a>
+            </div>
+            <div class="contact">
+                <a href="#">Contact</a>
+            </div>
+            <div class="mail">
+                <a href="#">
+                    <i class="fa fa-envelope-o"></i>
+                </a>
+            </div>
+
+        </nav>
+
+    </header>
+
+    <section class="header">
+        <h1>Patrick Maina</h1>
+        <h2>Software Developer</h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.
+        </p>
+        <div class="abilities-1">
+            <ul class="languages">
+                <li>Language</li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+
+            <ul class="frameworks">
+                <li>Frameworks</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+                
+            </ul>
+
+        </div>
+        <div class="abilities-2">
+            <ul class="skills">
+                <li>Skills</li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+                
+            </ul>
+
+        </div>
```

Comment:
- [ ] Optional.
Since the previous reviewer didn't raise this as a requirement, I am putting this as optional but strongly recommend you fix this so the layout of your application looks uniform and pleasant. It is probably a margin or padding issue.

| Figma design  | Your layout |
| ------------- | ------------- |
| ![figma](https://user-images.githubusercontent.com/33268927/170644575-6b5b9305-b611-442d-bb49-0632226a6f65.png) | ![padding-issue](https://user-images.githubusercontent.com/33268927/170644645-b977d4fc-29f2-4bc6-b45d-8c710b71ff45.png) |